Django/Oauth/README.md

```diff
@@ -0,0 +1,43 @@
+<p align="center"><a href="http://www.codechefvit.com" target="_blank"><img src="https://s3.amazonaws.com/codechef_shared/sites/all/themes/abessive/logo-3.png" title="CodeChef-VIT" alt="Codechef-VIT"></a>
+</p>
+
+# Project Title
+> OAuth-Pro
+---
+[![DOCS](https://img.shields.io/badge/Documentation-see%20docs-green?style=flat-square&logo=appveyor)](INSERT_LINK_FOR_DOCS_HERE) 
+  [![UI ](https://img.shields.io/badge/User%20Interface-Link%20to%20UI-orange?style=flat-square&logo=appveyor)](INSERT_UI_LINK_HERE)
+
+# Project Screenshorts
+
+[Here](https://www.linkpicture.com/q/google_oauth.png.png)
+ [Here][https://www.linkpicture.com/q/google_oauth.png1.png]
+[Here][https://www.linkpicture.com/q/google_oauth.png2.png]
+[Here][https://www.linkpicture.com/q/google_oauth.png3.png]
+[Here][https://www.linkpicture.com/q/google_oauth.png4.png]
+[Here][https://www.linkpicture.com/q/google_oauth.png5.png]
+[Final_results][https://www.linkpicture.com/q/google_oauth.png6.png]
```

Comment:
Hey it seems the images have not been rendered correctly ,can you look into this

Reply:
Sure